### ISSUE TITLE
"!" in docker-compose file leads to parsing error

### ISSUE THREAD

**Issue by unherz** [api-platform/api-platform#1982]
**API Platform version(s) affected**: 2.6.5

**Description**  
All lines containing "!ChangeMe!" as default value in the file _docker-compose.yml_ are leading to parsing errors like "yaml: line 37: did not find expected tag URI"

**How to reproduce**  
`docker-compose build --pull --no-cache`  after fresh installation

**Possible Solution**  
Replace all occurrences of "!ChangeMe!" with "ChangeMe"
or simply wrap the "!ChangeMe!" with quotes (not tested)

**Additional Context**  
Ref: https://cloudslang-docs.readthedocs.io/en/v1.0/yaml_overview.html#scalars-strings-numbers-values

Not 100% sure if this might be an edge case and/or specific for some system/setup combos
Basically this 

my setup:
WIN 10 2004 (Build 19041.1110)
Docker desktop
Docker Engine: 20.10.7
WSL 2

**Comment by Autsider666** [comment]
I'm getting the same error and nothing seems to work. The weird part is that I didn't have any issues with it this weekend.

**Comment by Autsider666** [comment]
@unherz I fixed it.... Forgot to turn off the experimental Docker Compose V2 feature. It all works now

**Comment by dunglas** [comment]
Does it change something to enclose the values with quotes?

**Comment by soyuka** [comment]
Closing per #2987 — this repository is becoming a CLI installer. The previous Docker/template distribution is being superseded by the installer + upstream [symfony-docker](https://github.com/dunglas/symfony-docker). If the issue still reproduces with the new setup, please open it on `dunglas/symfony-docker` (for Docker/Caddy/FrankenPHP issues) or `api-platform/core` (for framework issues).
